src/features/content-explorer/content-explorer/ContentExplorerFolderTreeBreadcrumbs.js

```diff
@@ -0,0 +1,64 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { injectIntl } from 'react-intl';
+
+import Button from '../../../components/button';
+import DropdownMenu from '../../../components/dropdown-menu/DropdownMenu';
+import IconBreadcrumbArrow from '../../../icons/general/IconBreadcrumbArrow';
+import IconFolderTree from '../../../icons/general/IconFolderTree';
+import { Menu, MenuItem } from '../../../components/menu';
+
+import { FoldersPathPropType } from '../prop-types';
+import messages from '../messages';
+import { bdlGray50 } from '../../../styles/variables';
+
+const ContentExplorerFolderTreeBreadcrumbs = ({
+    foldersPath,
+    intl: { formatMessage },
+    isFolderTreeButtonDisabled = false,
+    numTotalItems,
+    onBreadcrumbClick,
+}) => (
+    <div className="content-explorer-folder-tree-container">
+        <DropdownMenu>
+            <Button
+                aria-label={formatMessage(messages.clickToGoBack)}
+                className="content-explorer-folder-tree-button"
+                type="button"
+                title="file path"
```

Comment:
It was suggested by a technical writer that we should this message on hover. Whether it has to be a browser Tooltip or not is up for debate. I can ask my design team and see what their opinion is on it

Reply:
I spoke to my designer and they think that we should have a tooltip on the folder tree button icon because it is a new feature. They also like that the browser tooltip is more subtle than the Box tooltip, which seems better for something like this. Is this ok? Or should we not use the browser tooltip at all?

Reply:
yeah the browser tooltip should be okay but this should be localized.

did the designer want it to be all lower case though? `File path` seems more appropriate

Reply:
Gotcha I will localize that, and I will ask the designer what is best

Reply:
Yup, they said that "File path" is best. Sentence case